### DISCUSSION TITLE
No puedo loguear en los notebook del curso

### DISCUSSION THREAD

**Discussion by AndresGUIO2** [rramosp/ai4eng.v1#322] [01 - INTRODUCCIÓN]
Hola, adjunto la evidencia.

![image](https://github.com/user-attachments/assets/17a6bb4e-ce39-4ec4-9e75-f1a0922cb81f)

Ese error me aparece en cualquier laboratorio. 
Agradecería una pronta respuesta

**Comment by maikol0629** [comment]
Estoy teniendo el mismo problema

**Reply by admaesmo** [reply]
Ingresen con correo y contraseña directo sin usar el servicio de logueo de google. osea pidan recuperar contraseña en la plataforma y luego en el colab ingresan copn user y contraseña sin pedir token de google

**Comment by alejandr770** [comment]
tengo el mismo problema, ya habia solucionado este Lab, pero al no dejarme iniciar en el 2.03 intente con este y da el mismo error
![Captura de pantalla 2024-08-30 135708](https://github.com/user-attachments/assets/9386f3c9-8f0a-439b-82d1-9881c4045449)

**Reply by admaesmo** [reply]
hay problemas con la API de logueo de google ingrse directamente usuario y contraseña, yo solucioné así

**Comment by AlejandroOrregoRoldan** [comment]
Tengo el mismo problema
![image](https://github.com/user-attachments/assets/47432417-634c-4509-bfe5-3cd12e2d5cf8)

**Reply by admaesmo** [reply]
![WhatsApp Image 2024-08-31 at 10 09 50 PM](https://github.com/user-attachments/assets/d84496a7-d415-4031-90f2-a31f8c64384f)
![WhatsApp Image 2024-08-31 at 10 09 50 PM](https://github.com/user-attachments/assets/5314c436-ff12-4cdc-8e34-ec9a1f515dc8)
![WhatsApp Image 2024-08-31 at 10 09 50 PM(1)](https://github.com/user-attachments/assets/44ea32a6-77d7-49aa-aa66-a341cc694983)
![WhatsApp Image 2024-08-31 at 10 09 51 PM](https://github.com/user-attachments/assets/11547152-b1e1-4ee2-a9e8-9e8a43e64370)
![WhatsApp Image 2024-08-31 at 10 09 51 PM(1)](https://github.com/user-attachments/assets/b50d0ff6-7f7c-43de-ab82-af7c21ad554a)

**Comment by LauraMontoya994** [comment]
Cordial saludo profesor y compañeros. 

Al igual que los compañeros, tengo el mismo inconveniente para registrar mis respuestas en el Notebook. Agradezco inmensamente si nos ayudan con este inconveniente, pues entre hoy y mañana pensaba terminar los últimos tres laboratorios del Modulo 2 que me falta. 

De antemano, muchas gracias.

![image](https://github.com/user-attachments/assets/bfeb71ef-32a7-4a74-b69b-b0ebb88c1773)

**Comment by Marcesu2** [comment]
Hola compañeros y profesor, a mi me pasa el mismo problema por lo que no he podido enviar los intentos de laboratorios, si alguno ya tiene una solución  le agradezco si la comparte, gracias y estoy pendiente de cualquier novedad
![image](https://github.com/user-attachments/assets/133ba10b-f44f-4612-a452-ec727b04bff0)

**Comment by AndresGUIO2** [comment]
Para quienes tengan el problema, ejecutar esta instrucción en una celda de colab parece permitir el  logueo, aunque de momento no he enviado ningún laboratorio para comprobar que todo esté bien.

![image](https://github.com/user-attachments/assets/17f94a2e-2238-410e-9cde-5280732e163f)

**Comment by Rodas011** [comment]
Buenas, tengo el mismo error. Hace unos días pude realizar los laboratorios del módulo 1 sin problemas, pero ahora no permite el inicio de sesión. La solución que dio el compañero más arriba no funciona.
maira.rodas@udea.edu.co
![image](https://github.com/user-attachments/assets/41ba481c-a181-413d-8409-5ca6cb604cdb)

**Comment by Geckomonc** [comment]
Buenas noches, tengo el mismo error, no puedo ingresar escribiendo mi correo y contraseña ni tampoco utilizando el botón de Login with Google, por lo cual no puedo subir ningún laboratorio. 
Geraldine.acevedo@udea.edu.co
![image](https://github.com/user-attachments/assets/09cc6927-6afd-481b-8da0-bf1d4bc0baeb)
![image](https://github.com/user-attachments/assets/7c58ea2a-3e3c-497e-b4c2-6d8e9c72fd85)

**Comment by RickContreras** [comment]
Estoy teniendo problemas al ejecutar el siguiente código en el laboratorio 02.03 de Pandas:

```python
from local.lib.rlxmoocapi import submit, session
session.LoginSequence(endpoint=init.endpoint, course_id=init.course_id, lab_id="L02.03", varname="student")
```

Obtengo el siguiente error:

![Problema](https://github.com/user-attachments/assets/fe6a244f-8c11-4eba-954d-106c4dd911bc)

**Comment by M-Kleber** [comment]
Estoy teniendo problemas al ejecutar el siguiente código en el laboratorio LAB 02.01 - Python, en lab 02.02 y lab 02.04

![image](https://github.com/user-attachments/assets/7a74993f-737f-46cd-8cc8-ff26f1734682)

![image](https://github.com/user-attachments/assets/e1d9127a-6d0d-4c82-8a86-575bb45b9134)

![image](https://github.com/user-attachments/assets/f81fd279-8ef2-4302-93e2-47e334760d75)

**Comment by MateoBlandonNRA** [comment]
![Screenshot_4](https://github.com/user-attachments/assets/283a6c61-26f0-4f92-92ae-0a6139e4159c)
Saludos. Tengo el mismo problema, no puedo iniciar sesión.

**Reply by admaesmo** [reply]
Para los que están teniendo problemas para enviar los laboratorios, vayan al curso y de ahí a la "plataforma de seguimiento", denle debajo de login with email a "change/reset password", ingresen el correo de la u y sigan las instrucciones del correo que les llega para crear una contraseña. Ya deberían poder ingresar usando correo y contraseña, y enviar los laboratorios. El botón de Google no está funcionando y si solo se ha ingresado con el botón hay que asignarle una contraseña al correo para poder ingresar de la otra forma.

![WhatsApp Image 2024-08-31 at 10 09 50 PM](https://github.com/user-attachments/assets/6a8b335e-ef4f-4a60-bc47-029ec613a090)
![WhatsApp Image 2024-08-31 at 10 09 50 PM(1)](https://github.com/user-attachments/assets/86665eb6-574f-427c-ab74-31df07e429c1)
![WhatsApp Image 2024-08-31 at 10 09 51 PM](https://github.com/user-attachments/assets/a5892853-916a-492e-9ed0-2f1de8458e4c)
![WhatsApp Image 2024-08-31 at 10 09 51 PM(1)](https://github.com/user-attachments/assets/ad2caf85-3a41-4cfe-8d13-5bee70f46f86)

**Comment by Crudino** [comment]
Buenos días profe, tambien tengo el mismo problema que los compañeros no me deja ingresar. 
![Captura](https://github.com/user-attachments/assets/a5785db8-e6f8-4d68-82f8-0e051c7bd610)

**Comment by Helenfonnegra** [comment]
Buenos dias tambien tengo el mismo error de mis compañeros
![image (1)](https://github.com/user-attachments/assets/d0eb9459-8a6b-480e-ba34-ee2cdc119ae3)
![image](https://github.com/user-attachments/assets/a8cde808-1a0e-4db4-be5b-972f54d04082)

**Comment by JSebastianToro** [comment]
![Imagen de WhatsApp 2024-08-31 a las 13 22 52_cfb78562](https://github.com/user-attachments/assets/03bbad05-fd32-4767-9fee-3aeb3cb8be81)

Buenas tardes profe, yo también presento el mismo inconveniente que los compañeros al iniciar sección en los laboratorios.

**Comment by AlejandroOrregoRoldan** [comment]
Hola compañeros y profesor, a mi me pasa el mismo problema por lo que no he
podido enviar los intentos de laboratorios, si alguno ya tiene una solución
le agradezco si la comparte, gracias y estoy pendiente de cualquier
novedad, intente la solución del compañero más arriba pero no funciona.

El sáb, 31 ago 2024 a las 13:26, JSebastianToro ***@***.***>)
escribió:

> Imagen.de.WhatsApp.2024-08-31.a.las.13.22.52_cfb78562.jpg (view on web)
> <https://github.com/user-attachments/assets/03bbad05-fd32-4767-9fee-3aeb3cb8be81>
>
> Buenas tardes profe, yo también presento el mismo inconveniente que los
> compañeros al iniciar sección en los laboratorios.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/322#discussioncomment-10508321>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BK5DNBJFY5DOWUQQ6XBKZ5TZUIDEVAVCNFSM6AAAAABNM4VUESVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANJQHAZTEMI>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 


"La información aquí contenida es para uso exclusivo de la persona o 
entidad de destino. Está estrictamente prohibida su utilización, copia, 
descarga, distribución, modificación y/o reproducción total o parcial, sin 
el permiso expreso de Universidad de Antioquia, pues su contenido puede ser 
de carácter confidencial y/o contener material privilegiado. Si usted 
recibió esta información por error, por favor contacte en forma inmediata a 
quien la envió y borre este material de su computador. Universidad de 
Antioquia no es responsable por la información contenida en esta 
comunicación, el directo responsable es quien la firma o el autor de la 
misma."


-- 
UdeA

**Reply by admaesmo** [reply]
Para los que están teniendo problemas para enviar los laboratorios, vayan al curso y de ahí a la "plataforma de seguimiento", denle debajo de login with email a "change/reset password", ingresen el correo de la u y sigan las instrucciones del correo que les llega para crear una contraseña. Ya deberían poder ingresar usando correo y contraseña, y enviar los laboratorios. El botón de Google no está funcionando y si solo se ha ingresado con el botón hay que asignarle una contraseña al correo para poder ingresar de la otra forma.
![WhatsApp Image 2024-08-31 at 10 09 50 PM](https://github.com/user-attachments/assets/b6d80c70-86f8-4832-813b-ec8bcda761ff)
![WhatsApp Image 2024-08-31 at 10 09 50 PM(1)](https://github.com/user-attachments/assets/2cf0bca9-b420-409f-99ef-e7ef90a2c70e)
![WhatsApp Image 2024-08-31 at 10 09 51 PM](https://github.com/user-attachments/assets/6311744d-d99c-41e6-9989-fde93a491404)
![WhatsApp Image 2024-08-31 at 10 09 51 PM(1)](https://github.com/user-attachments/assets/0c1f4b13-8817-4f65-badc-5ecb0e2335b3)

**Comment by Felipe-RA** [comment]
Saludos a todos,

Ya estamos enterados del problema y le estamos buscando solución lo más pronto posible.

Gracias por vuestra paciencia

**Comment by Azzotea** [comment]
Buenas noches!,

También tengo este problema, lo intenté en la mañana y no funcionaba, por lo que esperé hasta la noche para ver si se solucionaba, pero todavía sigue el error

![error](https://github.com/user-attachments/assets/4e70750f-3c9c-499e-951b-d777fe0092d5)

**Comment by Rodas011** [comment]
Para los que están teniendo problemas para enviar los laboratorios, intenten si así les funciona: vayan al curso y de ahí a la "plataforma de seguimiento", denle debajo de login with email a "change/reset password", ingresen el correo de la u y sigan las instrucciones del correo que les llega para crear una contraseña. Ya deberían poder ingresar usando correo y contraseña, y enviar los laboratorios. El botón de Google no está funcionando y si solo se ha ingresado con el botón hay que asignarle una contraseña al correo para poder ingresar de la otra forma.
![image](https://github.com/user-attachments/assets/167271cc-877b-4c5c-a908-213dd1d97a08)
![image](https://github.com/user-attachments/assets/9fd5594b-ba7a-49a0-ac21-beeb9f1f2072)
![image](https://github.com/user-attachments/assets/0a5fdcfd-fc0d-4bda-82df-be164bdd2e6f)
![image](https://github.com/user-attachments/assets/5f0b03f5-5d59-471a-bf6b-ad33e9a1a257)
